api/config/laratrust.php

```diff
@@ -0,0 +1,344 @@
+    |
+    | The key in the array is the name of the relationship inside the roles and permissions.
+    |
+    */
+    'user_models' => [
+        'users' => \App\Models\User::class,
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laratrust Models
+    |--------------------------------------------------------------------------
+    |
+    | These are the models used by Laratrust to define the roles, permissions and teams.
+    | If you want the Laratrust models to be in a different namespace or
+    | to have a different name, you can do it here.
+    |
+    */
+    'models' => [
+
+        'role' => \App\Models\Role::class,
+
+        'permission' => \App\Models\Permission::class,
+
+        /**
+         * Will be used only if the teams functionality is enabled.
+         */
+        'team' => \App\Models\Team::class,
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laratrust Tables
+    |--------------------------------------------------------------------------
+    |
+    | These are the tables used by Laratrust to store all the authorization data.
+    |
+    */
+    'tables' => [
+
+        'roles' => 'roles',
+
+        'permissions' => 'permissions',
+
+        /**
+         * Will be used only if the teams functionality is enabled.
+         */
+        'teams' => 'teams',
+
+        'role_user' => 'role_user',
+
+        'permission_user' => 'permission_user',
+
+        'permission_role' => 'permission_role',
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laratrust Foreign Keys
+    |--------------------------------------------------------------------------
+    |
+    | These are the foreign keys used by laratrust in the intermediate tables.
+    |
+    */
+    'foreign_keys' => [
+        /**
+         * User foreign key on Laratrust's role_user and permission_user tables.
+         */
+        'user' => 'user_id',
+
+        /**
+         * Role foreign key on Laratrust's role_user and permission_role tables.
+         */
+        'role' => 'role_id',
+
+        /**
+         * Role foreign key on Laratrust's permission_user and permission_role tables.
+         */
+        'permission' => 'permission_id',
+
+        /**
+         * Role foreign key on Laratrust's role_user and permission_user tables.
+         */
+        'team' => 'team_id',
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Laratrust Middleware
+    |--------------------------------------------------------------------------
+    |
+    | This configuration helps to customize the Laratrust middleware behavior.
+    |
+    */
+    'middleware' => [
+        /**
+         * Define if the laratrust middleware are registered automatically in the service provider
+         */
+        'register' => true,
```

Comment:
I don't think we're planning to use Laratrust middleware on api routes, since we'll just be guarding the schema with Policies, so we can set this to false. (Having one less middleware might have an infinitesimal performance gain 😉 )